src/Workspaces/Core/Portable/Log/WorkspaceErrorLogger.cs

```diff
@@ -21,7 +21,7 @@ public WorkspaceErrorLogger()
         }
 
         public void LogException(object source, Exception exception)
-            => Logger.GetLogger()?.Log(FunctionId.Extension_Exception, LogMessage.Create(source.GetType().Name + " : " + ToLogFormat(exception), LogLevel.Error));
+            => Logger.Log(FunctionId.Extension_Exception, LogMessage.Create(source.GetType().Name + " : " + ToLogFormat(exception), LogLevel.Error));
```

Comment:
❗  This should be updated to use a delegate to avoid evaluation of `source.GetType().Name + " : " + ToLogFormat(exception)` when the logger is disabled.

Reply:
Done. Note that this only occurs when reporting unexpected exception, so it doesn't really matter.